apps/web/pages/organizations/[organizationId]/invoices/[invoiceId]/components/sendInvoiceButton.tsx

```diff
@@ -0,0 +1,145 @@
+import { ErrorMessage } from '@hookform/error-message'
+import { zodResolver } from '@hookform/resolvers/zod'
+import { format, isValid, isAfter, parseISO } from 'date-fns'
+import { useRef } from 'react'
+import { Controller, useForm } from 'react-hook-form'
+import InputMask from 'react-input-mask'
+import { z } from 'zod'
+
+import { invoiceSendInputValidations } from '@progwise/timebook-validations'
+
+import { CalendarSelector } from '../../../../../../frontend/components/calendarSelector'
+import { dateStringValidation, getDate } from '../../../../../../frontend/components/dateStringValidation'
+import { FragmentType, graphql, useFragment } from '../../../../../../frontend/generated/gql'
+import { InvoiceSendInput } from '../../../../../../frontend/generated/gql/graphql'
+
+export const SendInvoiceButtonFragment = graphql(`
+  fragment SendInvoiceButton on Invoice {
+    id
+    customerName
+    organization {
+      id
+    }
+  }
+`)
+
+export const InvoiceSendInputSchema: z.ZodSchema<InvoiceSendInput> = invoiceSendInputValidations.extend({
+  sendDate: z
+    .string()
+    .min('____-__-__'.length, 'Enter a date')
+    .refine((value) => value !== '____-__-__', 'Enter a date')
+    .refine((value) => !value || isValid(parseISO(value)), 'Invalid date')
+    .refine((value) => !value || !isAfter(parseISO(value), new Date()), 'Send date cannot be in the future'),
```

Comment:
```suggestion
    .refine((value) => !value || !isAfter(parseISO(value), new Date()), 'Send date cannot be a future date. Please enter a valid date.'),
```
Since I assume this will often be visible to a user, let's make it more user-friendly